terraform/modules/magine/variables.tf

```diff
@@ -28,3 +28,8 @@ variable "timeout" {
   description = "The amount of time your Lambda Function has to run in seconds"
   default     = 3
 }
+
+variable "runtime" {
+  description = "Lambda function runtime, nodejs8.10 or nodejs10.x"
+  default     = "nodejs8.10"
```

Comment:
shouldn't this be nodejs10.x?

Reply:
We're currently running 8.10, going to handle upgrading to 10 in a different PR

Reply:
@KempfCreative looks like it worked fine!